README.md

```diff
@@ -46,12 +46,18 @@ To hack on the `iohk-ops` tool, use
     io is a function
     io ()
     {
-        runhaskell -iiohk iohk/iohk-ops.hs "$@"
+        cabal exec iohk-ops -- "$@"
     }
     [nix-shell:~/iohk/iohk-ops]$ io --help
 
 This will provide a Haskell environment where you can use `io` to run
-the script or `ghci` for development.
+the script or `ghci` for development:
+
+    [nix-shell:~/iohk/iohk-ops]$ ghci -iiohk/common
```

Comment:
you can also use `cabal repl` or `runhaskell Setup.hs repl` to get ghci against a cabal project
i also often do `ghcid -c "runhaskell Setup.hs repl"` to get constant updates about errors

Reply:
Yes I would prefer `cabal repl` but it needs to be `ghci` because of the iohk-ops working directory thing. If I type `:cd` in `cabal repl` then it wants to reload all the modules.